ts/parts/Pointer.ts

```diff
@@ -1716,6 +1725,68 @@ Highcharts.Pointer.prototype = {
 
     },
 
+    /**
+     * @private
+     * @param {Highcharts.PointerEventObject} e
+     * @return {boolean}
+     */
+    isStickyTooltip: function (this: Highcharts.Pointer, e: Highcharts.PointerEventObject): boolean {
```

Comment:
Could you add a description what method actually does? From `Pointer.isStickyTooltip(event)` method name I expect something quite simple (compare for example with `chart.is3d()` or `series.is('seriesType')`, but it checks also DOM and position (?).

Reply:
This should be solved with the added description. Let me know if something is unclear.

Reply:
Aaah! Yes, that's clear. Thank you! :)